cosmic-settings/src/pages/desktop/mod.rs

```diff
@@ -9,18 +9,24 @@ pub mod window_management;
 pub mod workspaces;
 
 use cosmic::{config::CosmicTk, cosmic_config::CosmicConfigEntry};
+use cosmic_config::{ConfigGet, ConfigSet};
 use cosmic_settings_page as page;
+use tracing::error;
 
 #[derive(Debug)]
 #[allow(clippy::struct_excessive_bools)]
 pub struct Page {
-    pub cosmic_config: Option<cosmic::cosmic_config::Config>,
+    pub cosmic_tk_config: Option<cosmic::cosmic_config::Config>,
     pub cosmic_tk: CosmicTk,
```

Comment:
Not necessary to store this here because it's already cached and updated on config changes at `cosmic::config::COSMIC_TK`.

Reply:
ooh, nice!